htdocs/langs/fr_FR/other.lang

```diff
@@ -35,6 +35,9 @@ OnlyOneFieldForXAxisIsPossible=1 seul champ est actuellement possible en tant qu
 AtLeastOneMeasureIsRequired=Au moins 1 champ de mesure est requis
 AtLeastOneXAxisIsRequired=Au moins 1 champ pour l'axe X est requis
 LatestBlogPosts=Derniers articles du Blog
+notiftouser=Aux utilisateurs
```

Comment:
Do not modify language files that are not en_US. Changes will be erases by transifex sync.

Reply:
ok i have delet